pkg/handlers/deploy.go

```diff
@@ -85,19 +85,14 @@ func (h *Handler) DeployAppVersion(w http.ResponseWriter, r *http.Request) {
 	// preflights reports
 	if !request.IsAirgap {
 		go func() {
+			isCli := false
```

Comment:
i don't agree with this pattern. this is a handler, and can be called from anywhere. deciding if it's a cli or not should not be made here in the api.